build/yamls/flow-aggregator.yml

```diff
@@ -219,8 +217,14 @@ spec:
           name: flow-aggregator-config
           readOnly: true
           subPath: flow-aggregator.conf
+        - mountPath: /var/log/antrea
+          name: host-var-log-antrea
       serviceAccountName: flow-aggregator
       volumes:
       - configMap:
           name: flow-aggregator-configmap-hf78268hm6
         name: flow-aggregator-config
+      - hostPath:
+          path: /var/log/antrea
```

Comment:
Probably better to have a sub dir for flow aggregtor, as the same log dir can be used for other components too (agent/controller/ovs).

Reply:
I have no strong preference here. The agent and controller logs are already in the same directory, and the component name is part of the log file name. I am fine with a sub directory though.

Reply:
Yes, but today only agent/controller logs are in the top dir; OVS is not. I prefer a sub-dir for flow aggregator too.

Reply:
Thanks, updated the log directory to `/var/log/antrea/flow-aggregator`